src/shared/modules/commands/helpers/params.js

```diff
@@ -42,14 +72,33 @@ export const handleParamsCommand = (action, cmdchar, put, store) => {
     } else {
       // Single param
       try {
-        const json = '{' + param + '}'
-        const res = jsonic(json)
-        put(update(res))
-        return resolve({ result: res, type: 'param' })
+        const { key, value, isFn } = extractParams(param)
+        if (!key || !value || !isFn) {
+          return resolveAndStoreJsonValue(param, put, resolve, reject)
+        }
+        const cypherStatement = mapParamToCypherStatement(key, value)
+        bolt
+          .routedWriteTransaction(
+            cypherStatement,
+            {},
+            {
+              useCypherThread: false,
+              requestId: action.requestId,
+              cancelable: false
+            }
+          )
+          .then(res => {
+            let obj = {}
+            res.records.forEach(record => {
+              obj[key] = record.get(key)
+            })
+            const result = recursivelyTypeGraphItems(obj)
+            put(update(result))
+            resolve({ result, type: 'param' })
+          })
+          .catch(e => reject(e))
       } catch (e) {
-        return reject(
-          new Error('Could not parse input. Usage: `:param "x": 2`. ' + e)
-        )
+        reject(new Error('Could not parse input. Usage: `:param "x": 2`. ' + e))
```

Comment:
I think we should promote the new format here

Reply:
Display both usages or only the new `=>` approach?

Reply:
Both, I think.